### ISSUE TITLE
[Usability]A11y_Manage.IIS.Net_Web Server_IP Restrictions_Screen Reader: NVDA is not announcing the number/level of the list item on which the focus is(example: NVDA should read as 1 of 4)

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#176]
GitHubTags: #A11y_Manage.IIS.Net; #A11yUsable; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #ScreenReader; #Chrome; #NVDA;

**Environment Details:**
URL: https://manage.iis.net/webserver/ip-restrictions
Browser Details: Google Chrome 68.0.3440.106
Screen Reader: NVDA

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till IP Restrictions and press enter
6. Tab till "Response Status When Denied" combo box and press enter. Verify NVDA is announcing number/level of the list item on which the focus is

**Actual Result:**
NVDA is not announcing number/level of the list item on which the focus is

**Expected Result:**
NVDA should announce the number/level of the list item on which the focus is

**Note:**
NVDA is also not reading the levels (such as if there are 4 items in a combo box, if focus is on 2nd list item it should read as 2 of 4)

**User Impact:**
When NVDA announce the number/level of the list item on which the focus is it would be more user friendly to the screen reader users

**Attachments:**
[Bug176_web server_ip restrictions_nvda not announcing the no of list items.pptx](https://github.com/Microsoft/IIS.WebManager/files/2335328/Bug176_web.server_ip.restrictions_nvda.not.announcing.the.no.of.list.items.pptx)

**Comment by jhkimnew** [comment]
Seems to be fixed.
